Tringo.WebApp/Controllers/FlightsController.cs

```diff
@@ -69,25 +69,32 @@ public async Task<ActionResult<IEnumerable<FlightDestinationResponse>>> GetDesti
             // just for testing and must be deleted soon
             await Task.Delay(100); // TODO: remove this line
 
-            if (inputData?.Dates == null)
+            if (inputData?.Dates == null
+                || inputData.SearchArea == null
+                || inputData.Budget == null)
                 return new BadRequestResult();
```

Comment:
Controller should not be checking the contract
Put Data Annotations members

https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations?view=netcore-2.2

Reply:
updated